.circleci/config.yml

```diff
@@ -134,7 +134,7 @@ executors:
     parameters:
       resource_class:
         type: string
-        default: large
+        default: xlarge
```

Comment:
Do we really need to do this. xlarge is quite expensive. When we are just running smoke tests, we don't run the fxa stack in the CI pipeline, so it doesn't _seem_ like we should need a super big instance.

Reply:
As of 5 months ago, every functional test job, except smoke production, has already been running on an xlarge instances. Since the production smoke tests run so infrequently I think this generalization is probably negligible.
I did inquire about costs when we first moved the nightly and pr jobs to xlarge and was reassured it wasn't a concern by SRE.